utils.py

```diff
@@ -70,7 +70,7 @@ def _get_vhost_domains(self, vhost_name, data):
     def _parse_vhosts_pl_section(self, vhosts_pl_path, section):
         return json.loads(subprocess.check_output([
             'perl', '-e', 'use JSON; require "' + vhosts_pl_path + '"; print encode_json($' + section + ');'
-        ]))
+        ]).decode('utf-8'))
```

Comment:
I realise we're on 3.5, just to note that from python 3.6 json.loads() does accept bytes here.